database/src/backend/redb/database.rs

```diff
@@ -9,38 +9,44 @@ use crate::{
 };
 
 //---------------------------------------------------------------------------------------------------- DatabaseRo
-impl<T: Table> DatabaseRo<T> for RedbTableRo<'_> {
-    fn get(&self, key: &T::Key) -> Result<Option<T::Value>, RuntimeError> {
+impl<T: Table> DatabaseRo<T> for RedbTableRo<'_, T::Key, T::Value> {
+    fn get(&self, key: &T::Key) -> Result<Option<&T::Value>, RuntimeError> {
```

Comment:
I just realized this, I don't think we need the `Result<Option<_>,_>` return value only the result as `RuntimeError` contains an error for `KeyNotFound`